src/Adapter/MSTestAdapter.PlatformServices/Services/ThreadSafeStringWriter.cs

```diff
@@ -181,6 +181,14 @@ private ThreadSafeStringBuilder GetOrAddStringBuilder()
         }
     }
 
+    internal static void CleanState()
```

Comment:
I don't see this used anywhere?

Reply:
It's used in TestMethodInfo.Invoke to separate potential assembly init or class init from the methods.